src/Groups/abelian_aut.jl

```diff
@@ -268,3 +268,137 @@ function embedding_orthogonal_group(i::TorQuadModuleMor)
   return OAtoOD::GAPGroupHomomorphism{AutomorphismGroup{TorQuadModule}, AutomorphismGroup{TorQuadModule}}
 end
 
+###############################################################################
+#
+#  Action on injections
+#
+###############################################################################
+
+@doc raw"""
+    is_invariant(f::TorQuadModuleMor, i::TorQuadModuleMor) -> Bool
+
+Given an abelian group embedding $i\colon S \to T$ of a torsion quadratic module
+`S` in a torsion quadratic module `T`, and an abelian group endomorphism `f`
+of `T`, return whether `f` fixes `S`, i.e. whether $f(i(s)) \in i(S)$ for all
+$s \in S$.
+"""
+function is_invariant(f::TorQuadModuleMor, i::TorQuadModuleMor; check::Bool = true)
+  @req !check || is_invariant(i) "i must be an injection"
```

Comment:
Mathematically, it makes sense also for `i` not injective.

Reply:
Yes  indeed, if we talk about invariance of the image. The idea for `is_invariant` was really related to invariant subgroups. I could remove the restriction though, but will it be of any use ?

Reply:
Can't hurt.